app/uk/gov/hmrc/servicecommissioningstatus/connectors/TeamsAndRepositoriesConnector.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.servicecommissioningstatus.connectors
+
+import uk.gov.hmrc.http.{HeaderCarrier, HttpReads, StringContextOps}
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+object TeamsAndRepositoriesConnector {
+  import play.api.libs.functional.syntax._
+  import play.api.libs.json._
+
+  sealed trait ServiceType {def asString: String}
+  object ServiceType {
+    case object Frontend extends ServiceType { def asString = "FrontendService" }
+    case object Backend extends ServiceType  { def asString = "BackendService"  }
+
+    val values = Set(Frontend, Backend)
+
+    def parse(s: String): Either[String, ServiceType] =
+      values
+        .find(_.asString.equalsIgnoreCase(s))
+        .toRight(s"Invalid serviceType - should be one of: ${values.map(_.asString).mkString(", ")}")
+
+    val read = new Reads[ServiceType] {
+      override def reads(js: JsValue): JsResult[ServiceType] =
+        js.validate[String]
+          .flatMap(parse(_).fold(JsError(_), JsSuccess(_)))
+    }
+  }
+
+  sealed trait Tag {def asString: String}
+
+  object Tag {
+    case object AdminFrontend extends Tag { def asString = "AdminFrontend" }
+    case object Api           extends Tag { def asString = "Api"           }
+    case object Stub          extends Tag { def asString = "Stub"          }
+
+    val values = Set(AdminFrontend, Api, Stub)
+
+    def parse(s: String): Either[String, Tag] =
```

Comment:
This is a duplication of Line 37 buy the looks of it

Reply:
This is a copy of how we treat types in our other services such as Environment. It's easier to lift and re-use as we can't create a shared library.

Reply:
Made `StringType` trait to reduce duplicate code